src/api/Kandidat.ts

```diff
@@ -1,21 +1,21 @@
-import { ArbeidsmijøBehov, ArbeidstidBehov, FysiskBehov, GrunnleggendeBehov } from './Behov';
+import { Arbeidshverdagen, Arbeidstid, FysiskTilrettelegging, UtfordringerMedNorsk } from './Behov';
 
 export interface Kandidat {
     aktørId: string;
     fnr: string;
     sistEndret: string;
     sistEndretAv: string;
     navKontor: string;
-    arbeidstidBehov: ArbeidstidBehov[];
-    fysiskeBehov: FysiskBehov[];
-    arbeidsmiljøBehov: ArbeidsmijøBehov[];
-    grunnleggendeBehov: GrunnleggendeBehov[];
+    arbeidstidBehov: Arbeidstid[];
+    fysiskeBehov: FysiskTilrettelegging[];
+    arbeidsmiljøBehov: Arbeidshverdagen[];
```

Comment:
Keysa her må vel også endres

Reply:
Det er en egen oppgave som krever samkjøring mellom backend og frontend så vi tenkte å ta det for seg selv

Reply:
Smud!